sorts/double_sort.py

```diff
@@ -14,29 +17,32 @@ def double_sort(lst):
     >>> double_sort([-3, 10, 16, -42, 29]) == sorted([-3, 10, 16, -42, 29])
     True
     """
-    no_of_elements = len(lst)
+    no_of_elements = len(collection)
     for _ in range(
         int(((no_of_elements - 1) / 2) + 1)
     ):  # we don't need to traverse to end of list as
         for j in range(no_of_elements - 1):
             if (
-                lst[j + 1] < lst[j]
+                collection[j + 1] < collection[j]
             ):  # applying bubble sort algorithm from left to right (or forwards)
-                temp = lst[j + 1]
-                lst[j + 1] = lst[j]
-                lst[j] = temp
+                collection[j], collection[j + 1] = collection[j + 1], collection[j]
             if (
-                lst[no_of_elements - 1 - j] < lst[no_of_elements - 2 - j]
+                collection[no_of_elements - 1 - j] < collection[no_of_elements - 2 - j]
             ):  # applying bubble sort algorithm from right to left (or backwards)
-                temp = lst[no_of_elements - 1 - j]
-                lst[no_of_elements - 1 - j] = lst[no_of_elements - 2 - j]
-                lst[no_of_elements - 2 - j] = temp
-    return lst
+                (
+                    collection[no_of_elements - 1 - j],
+                    collection[no_of_elements - 2 - j],
+                ) = (
+                    collection[no_of_elements - 2 - j],
+                    collection[no_of_elements - 1 - j],
+                )
+    return collection
 
 
 if __name__ == "__main__":
-    print("enter the list to be sorted")
-    lst = [int(x) for x in input().split()]  # inputing elements of the list in one line
-    sorted_lst = double_sort(lst)
+    print("Enter the list to be sorted: ")
+    unsorted = [
+        int(x) for x in input().split() if x
+    ]  # inputing elements of the list in one line

```

Comment:
Let's not wrap lines of code just to make room for a trailing comment.
```suggestion
    # allow the user to input the elements of the list on one line
    unsorted = [int(x) for x in input("Enter the list to be sorted: ").split() if x]
```